src/toolkit/res_map.h

```diff
@@ -233,7 +233,7 @@ class ResMap {
   }
 
   /// Whether quantity_ should be recomputed or not.
-  bool dirty_quantity_;
+  mutable bool dirty_quantity_;
```

Comment:
This seems unrelated... any context?

Reply:
On newer compilers an error will be thrown by that line.  I think I originally installed the compilers explicitly and it was installing newer versions of gcc, but by simply installing `compilers` via conda it installs a slightly older version of gcc that doesn't throw the error.  I do think installing `compilers` instead of explicitly specifying `gxx_linux-64 gcc_linux-64` is the better strategy, since conda will find the appropriate compiler packages for you based on your system.  As time goes on we will likely eventually need to make this change (especially if I can get the macOS build to work), but for now we can leave it out since it's not necessary.

It first presented itself to me when working on the macOS and using clang.